README.md

```diff
@@ -199,6 +199,7 @@ Papers:
 - 2014, [Exploiting Ordered Dictionaries to Efficiently Construct Histograms with Q-Error Guarantees in SAP HANA](https://dl.acm.org/doi/pdf/10.1145/2588555.2595629), SIGMOD
 - 2017, [Adaptive Statistics in Oracle 12c](http://www.vldb.org/pvldb/vol10/p1813-zait.pdf), VLDB
 - 2019, [Pessimistic Cardinality Estimation: Tighter Upper Bounds for Intermediate Join Cardinalities](https://dl.acm.org/doi/pdf/10.1145/3299869.3319894), SIGMOD
+- 2019, [Every Row Counts: Combining Sketches and Sampling for Accurate Group-By Result Estimates](https://db.in.tum.de/~freitag/papers/p23-freitag-cidr19.pdf), CIDR
```

Comment:
how about "Hyperloglog: The analysis of a near-optimal cardinality estimation algorithm"?

Reply:
This CIDR paper is a successor of HyperLogLog and enables a updatable cardinality estimation. 
It fits more into database's query optimization context than the original HyperLogLog.

Reply:
got